specification/botservice/resource-manager/Microsoft.BotService/preview/2017-12-01/botservice.json

```diff
@@ -683,6 +683,57 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.BotService/locations/{location}/operationresults/{operationId}": {
+      "get": {
+        "x-ms-examples": {
+          "Get Server Details": {
+             "$ref": "./examples/operationResults.json"
+             }
+        },
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "OperationResult_Get",
+        "x-ms-long-running-operation": true,

```

Comment:
`GET` operations MUST not be long running. If the resource takes a long time to come up, please user provisioning states.

Reply:
i added the x-ms-longrunning attribute as autorest was complaining about it, so i have now removed it. 
instead of a long running operation we merely want a way to query for an operation result. the operation result operation isnt longrunning itself. sorry for the confusion.